packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_prop/independent_prop.rs

```diff
@@ -26,18 +26,20 @@ where
     T: Default + Clone + TryFrom<PropValue> + std::fmt::Debug + 'static,
     PropValue: From<T>,
     <T as TryFrom<PropValue>>::Error: std::fmt::Debug,
+    <T as TryFrom<PropValue>>::Error: std::fmt::Display,
```

Comment:
For clarify, use the same format as the below cases (lines 42, 51, etc.):
<T as TryFrom<PropValue>>::Error: std::fmt::Display + std::fmt::Debug,